src/discount/discounts.mock.ts

```diff
@@ -4,5 +4,6 @@ export function getDiscount(): Discount {
     return {
         id: '12e11c8f-7dce-4da3-9413-b649533f8bad',
         discountedAmount: 10,
+        discountedAmountAsInteger: 10,
```

Comment:
can we add correct values for the Integer values 10 => 1000, this applies to all the mocked values which represent integers